solutions/aspnet_core/aspnet_samples/Makefile

```diff
@@ -0,0 +1,64 @@
+TOP=$(abspath ../../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+
+ifdef STRACE
+OPTS += --strace
+endif
+
+TIMEOUT=30s
+
+build: rootfs
+
+appdir:
+	$(APPBUILDER) Dockerfile
+
+rootfs: appdir
+	$(MYST) mkext2 appdir rootfs
+
+clean: kill-db
+	./kill.sh FlightFinder
+	./kill.sh Todo
+	./kill.sh Podcast
+	rm -rf appdir rootfs client.output
+
+run: run-flight run-todo run-podcast
+
+run-flight:
+	echo "Running FlightFinder sample"
+	./kill.sh FlightFinder
+	timeout $(TIMEOUT) $(MYST_EXEC) $(OPTS) --app-config-path config-flight.json rootfs /usr/bin/dotnet /built/FlightFinder/FlightFinder.Server.dll &
+	timeout $(TIMEOUT) ./wait.sh
+	./client.sh FlightFinder
+	./kill.sh FlightFinder
+	test -f client.output
+
+run-todo:
+	echo "Running Todo sample"
+	./kill.sh Todo
+	timeout $(TIMEOUT) $(MYST_EXEC) $(OPTS) --app-config-path config-todo.json rootfs /built/TodoApi/TodoApi &
+	timeout $(TIMEOUT) ./wait.sh
+	./client.sh Todo
+	./kill.sh Todo
+
+run-podcast:
+	echo "Running podcast sample"
+	$(MAKE) kill-db
+	$(MAKE) run-podcast-inner
+	$(MAKE) kill-db
+
+run-podcast-inner: run-db
+	./kill.sh Podcast
+	timeout $(TIMEOUT) $(MYST_EXEC) $(OPTS) --app-config-path config-podcast.json rootfs /built/dotnet-podcasts/Podcast.API &
+	timeout $(TIMEOUT) ./wait.sh
+	./client.sh Podcast
+	./kill.sh Podcast
+
+# This is used by Podcast sample
+run-db:
+	docker run --name dotnet-podcast-db -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=Pass@word" -p 5433:1433 -d mcr.microsoft.com/mssql/server:2019-latest
```

Comment:
The best practice is to not embed passwords as clear text in the source code. Is there a way to use an environment variable instead?

Reply:
Not sure if that's necessary for a sample like this? To define this password as env variable here, it still going to be a plain text like `SA_PASSWORD=Password docker run <...>`
If we define this variable say only in Jenkin server, then it may not be obvious here how to run this sample

Reply:
It might trigger warning from `credscan` in the future. But for now I think we can accept an dummy password for a SQL server that runs only once.